specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -8394,7 +8437,15 @@
         }
       },
       "description": "LogAnalytics output properties"
-    }
+    },
+    "VMScaleSetConvertToSinglePlacementGroupInput": {
+      "properties":{
+        "activePlacementGroupId": {
+            "type": "string",
+            "description": "Id of the placement group in which you want future virtual machine instances to be placed. To query placement group Id, please use Virtual Machine Scale Set VMs - Get API. If not provided, the platform will choose one with maximum number of virtual machine instances."
```

Comment:
this is a required property? if so please mark it as required

Reply:
No, `activePlacementGroupId` is an optional parameter. If it's not present in customer's request, we'll internally select one as default if possible.